liota/entities/metrics/registered_metric.py

```diff
@@ -160,7 +160,7 @@ def send_data(self):
         """
         log.info("Publishing values for the resource {0} ".format(
             self.ref_entity.name))
-        if not self.values:
+        if self.values.qsize() == 0:
             # No values measured since last report_data
             return True
```

Comment:
Just an observation - send_data() returns True when there are no metrics queued. Otherwise, nothing is returned.
Should send_data() return True only if metric publish is successful and False, otherwise?

Reply:
Very good point. There is no method consuming this return value for now. Since it is part of existing code, will not change it in this pull request and will defer for next changelist.